src/Catalyst.Cli/Program.cs

```diff
@@ -54,17 +54,15 @@ static Program()
         /// <summary>
         ///     Main cli loop
         /// </summary>
-        public static int Main(string[] args)
+        public static async Task<int> Main(string[] args)
         {
             // Parse the arguments.
-            Parser.Default
-               .ParseArguments<Options>(args)
-               .WithParsed(async o => await RunAsync(o).ConfigureAwait(false));
-
-            return Environment.ExitCode;
+            var result = await Parser.Default.ParseArguments<Options>(args)
+               .MapResult(async options => await RunAsync(options), response => Task.FromResult(1));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/app/catalyst-network/Catalyst.Framework/pullRequest?prid=4699939)